docs/examples/endpoint/web.yaml

```diff
@@ -0,0 +1,19 @@
+name: web
+replicas: 1
+containers:
+- image: wordpress:4
+  env:
+  - name: WORDPRESS_DB_HOST
+    value: database:3306
+  - name: WORDPRESS_DB_PASSWORD
+    value: wordpress
+  - name: WORDPRESS_DB_USER
+    value: wordpress
+  - name: WORDPRESS_DB_NAME
+    value: wordpress
+services:
+- name: wordpress
+  ports:
+  - port: 8080
+    targetPort: 80
+    endpoint: minikube.local/foo
```

Comment:
remove `/foo`.
Wordpress is not configured to handle `/foo` so you end up with invalid redirect

Reply:
fixed